src/containers/PasswordReset/Passwordreset.js

```diff
@@ -0,0 +1,163 @@
+            const errors = error.response.data.error
+            this.setState({
+                errors: {
+                    "email": [errors]
+                },
+                resetbutton: {
+                    ...resetbutton,
+                    disabled: "disabled"
+                }
+            });
+        });
+    }
+
+    renderForm = () => {
+        const { errors, resetbutton } = this.state;
+
+        return (
+            <div data-test="renderform-test" className="row valign-wrapper vertical-center">
+                <div className="col s12 m4 offset-m4 valign">
+                    <div className="card">
+                        <form onSubmit={this.handleSubmit(apiCalls)}>
+                            <div className="card-content">
+                                <span className="card-title center-align">Forgot your Password?</span>
+                                <p className="center-align grey-text">
+                                    Type in the email address your registered with
+                                </p>
+                                <br />
+                                <div className="row">
+                                    {renderInput({ id: "reset-email", name: "email", errors, label: "Email", type: "email" }, this.handleChange)}
+                                </div>
+                            </div>
+                            <div className="card-action center-align">
+                                <input {...resetbutton} type="submit" className="btn orange" />
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </div>
+        )
+    }
+
+    renderSuccessMessage = () => {
+        const { user: { email } } = this.state
+        return (
+            <div className="container" data-test="rendermessage-test">
+                <div className="vertical-center">
+                    <div className="row" >
+                        <div className="col s6 offset-s3 center-align ">
+                            <div className="col s10 offset-s1">
+                                <div>
+                                    <i className="far fa-envelope fa-3x"></i>
+                                </div>
+                                <div>
+                                    <span><h5>Check your mail</h5></span>
+                                </div>
+                                <div className="">
+                                    <span className="grey-text">We have received your request to reset your password and sent a reset password link to {email}</span>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        );
+    }
+
+    render() {
+        const { status } = this.state;
+        return status ? this.renderSuccessMessage() : this.renderForm()
```

Comment:
Good job, I like how elegant this render method is. The functionality works as expected 👍